tests/consensus_tests/test_resharding.py

```
@@ -300,8 +300,9 @@ def test_resharding_down_abort_cleanup(tmp_path: pathlib.Path, peers: int):
     resp = abort_resharding(peer_uris[0])
     assert_http_ok(resp)
 
-    # Wait for resharding to abort
-    wait_for_collection_resharding_operations_count(peer_uris[0], COLLECTION_NAME, 0)
+    # Wait for resharding to abort on all peers
+    for peer_uri in replica_uris:
+        wait_for_collection_resharding_operations_count(peer_uri, COLLECTION_NAME, 0)
```

Ruff (0.15.7)

[error] `wait_for_collection_resharding_operations_count` may be undefined, or defined from star imports

(F405)